src/frontend/screens/Library/components/InstallModal/index.tsx

```diff
@@ -145,7 +146,12 @@ export default React.memo(function InstallModal({
       >
         {availablePlatforms.map((p, i) => (
           <MenuItem value={p.value} key={i}>
-            {p.name}
+            <Box sx={{ display: 'flex', placeItems: 'center' }}>
+              <SvgIcon sx={{ mr: 1 }}>
```

Comment:
I wonder if there's a way to set `margin-inline-end` instead of `margin-right` with these MUI syntax for styling, so it works for RTL languages

because we need the margin on the left for those

Reply:
MUI's `sx` prop is essentially a better `style` prop, so we can set any CSS property there